bundler/README.md

```diff
@@ -1,5 +1,5 @@
 [![Version     ](https://img.shields.io/gem/v/bundler.svg?style=flat)](https://rubygems.org/gems/bundler)
-[![Inline docs ](https://inch-ci.org/github/rubygems/bundler.svg?style=flat)](https://inch-ci.org/github/rubygems/bundler)
+[![Inline docs ](https://inch-ci.org/github/rubygems/rubygems.svg?style=flat)](https://inch-ci.org/github/rubygems/rubygems)
```

Comment:
This link may not be as relevant as before because it now applies to the whole RubyGems project, however, I’ve decided to just update it in this pull request.

Reply:
I'm not sure if anybody cares about inch-ci. And I'm also not sure whether we could configure it to manage docs for both projects in this repo.

But for now I wouldn't touch this since at least it makes it clear that the docs page is not updated since the repo merge. There's indeed a rubygems inch-ci page too, but [it hasn't been built since 4 years ago](https://inch-ci.org/github/rubygems/rubygems/history?branch=master) so that one I'm pretty sure nobody cares about.

Reply:
I’ve opened https://github.com/rubygems/rubygems/pull/3617 to remove the Inch CI badge.

I’d rather simply update the link in this pull request because keeping the old one could be confusing as it would always show an outdated evaluation. However, if you prefer, I can also omit this change.

Reply:
Ok, fair enough!